crates/common/src/casm_class.rs

```diff
@@ -0,0 +1,103 @@
+use num_bigint::BigUint;
+use num_traits::Num;
+use serde::{Deserialize, Serialize};
+
+/// A contract in the Starknet network.
+#[derive(Debug, Serialize, Deserialize)]
+pub struct CasmContractClass {
+    pub bytecode: Vec<BigUintAsHex>,
```

Comment:
Can't we just use `Vec<Felt>` here? I understand it's a BigUintAsHex in blockifier but I _think_ all these bigints are actually felts in here.

Reply:
Yeah I _felt_ ;) this would be brought up, will adjust!